tests/SprykerEcoTest/Zed/Braintree/_support/BraintreeBusinessTester.php

```diff
@@ -29,7 +37,77 @@ class BraintreeBusinessTester extends Actor
 {
     use _generated\BraintreeBusinessTesterActions;
 
-   /**
-    * Define custom actions here
-    */
+    /**
+     * @param array $params
+     *
+     * @return \Braintree\Transaction
+     */
+    public function getSuccessfulTransaction($params = []): Transaction
+    {
+        $defaultParams = [
+           'id' => 123,
+           'processorResponseCode' => '1000',
+           'processorResponseText' => 'Approved',
+           'createdAt' => new DateTime(),
+           'status' => 'authorized',
+           'type' => 'sale',
+           'amount' => 0,
+           'merchantAccountId' => 'abc',
+           'statusHistory' => new StatusDetails([
+               'timestamp' => new DateTime(),
+               'status' => 'authorized',
+           ]),
+        ];
+
+        return Transaction::factory(array_merge($defaultParams, $params));
+    }
+
+    /**
+     * @param array $params
+     *
+     * @return \Braintree\Result\Successful
+     */
+    public function getSuccessfulTransactionResponse($params = []): Successful
+    {
+        return new Successful($this->getSuccessfulTransaction($params));
+    }
+
+    /**
+     * @param bool $isSuccess
+     *
+     * @return \Generated\Shared\Transfer\BraintreeTransactionResponseTransfer
+     */
+    public function getBraintreeTransactionResponseTransfer(bool $isSuccess): BraintreeTransactionResponseTransfer
+    {
+        return (new BraintreeTransactionResponseTransfer())
+            ->setIsSuccess($isSuccess);
+    }
+
+    /**
+     * @param \Generated\Shared\Transfer\OrderTransfer $orderTransfer
+     *
+     * @return \Generated\Shared\Transfer\CheckoutResponseTransfer
+     */
+    public function getCheckoutResponseTransfer(OrderTransfer $orderTransfer): CheckoutResponseTransfer
+    {
+        $saveOrderTransfer = $this->createSaveOrderTransfer($orderTransfer);
+
+        $checkoutTransfer = new CheckoutResponseTransfer();
+        $checkoutTransfer->setSaveOrder($saveOrderTransfer);
+
+        return $checkoutTransfer;
+    }
+
+    /**
+     * @param \Generated\Shared\Transfer\OrderTransfer $orderTransfer
+     *
+     * @return \Generated\Shared\Transfer\SaveOrderTransfer
+     */
+    public function createSaveOrderTransfer(OrderTransfer $orderTransfer): SaveOrderTransfer
+    {
+        $saveOrderTransfer = new SaveOrderTransfer();
```

Comment:
use methods chaining if this method is needed at all

Reply:
Refactored. This method was removed.